charts/cluster-scanner/templates/deployment.yaml

```diff
@@ -97,6 +97,8 @@ spec:
               configMapKeyRef:
                 name: {{ include "cluster-scanner.fullname" . }}
                 key: cluster_name
+                optional: true
+          - name: SYSDIG_KUBECONFIG_CONTENT
```

Comment:
Is `- name: SYSDIG_KUBECONFIG_CONTENT` really required?

Reply:
😮 non sure why it's there, gonna fix soon